tiup/tiup-component-cluster-check.md

```diff
@@ -142,6 +143,14 @@ tiup cluster check <topology.yml | cluster-name> [flags]
 - Data type: `BOOLEAN`
 - This option is disabled by default with the `false` value. To enable this option, add this option to the command, and either pass the `true` value or do not pass any value.
 
+> **Note:**
+>
+> tiup-cluster also supports fixing the `scale-out yaml` file for a deployed cluster with the following command format:
```

Comment:
```suggestion
> tiup-cluster also supports repairing the `scale-out yaml` file for a deployed cluster with the following command format:
```
Line 136 uses the word "repair". Better to use the same word to avoid confusion.